router_test.go

```
@@ -396,6 +399,140 @@ func Test_App_Rebuild_Tree(t *testing.T) {
 	require.Equal(t, http.StatusOK, resp.StatusCode, "Status code")
 }
 
+func Test_App_Remove_Route(t *testing.T) {
+	t.Parallel()
+	app := New()
+
+	app.Get("/test", func(c Ctx) error {
```

golangci-lint (1.62.2)

[warning] empty-lines: extra empty line at the start of a block

(revive)